runtime/oti/j9consts.h

```diff
@@ -246,7 +246,7 @@ extern "C" {
 #define J9_RESOLVE_FLAG_FIELD_SETTER 0x10
 #define J9_RESOLVE_FLAG_CHECK_CLINIT 0x20
 #define J9_RESOLVE_FLAG_INSTANTIABLE 0x40
-#define J9_RESOLVE_FLAG_AOT_LOAD_TIME 0x80
+#define J9_RESOLVE_FLAG_UNUSED_FLAG1 0x80 /* This bit is available for re-use */
```

Comment:
Please call this `UNUSED_0x80`, and the comment is unnecessary.

Reply:
That was a spurious change due to an extra commit sneaking in.

Reply:
This change was already delivered in #6000